weblab/templates/datasets/compare_fittings.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+{% load entities %}
+{% load fittings %}
+{% load staticfiles %}
+
+{% block title %}Dataset : compare fitting experiments - {% endblock title %}
+
+{% block content %}
```

Comment:
There's a lot of overlap here with the dataset_detail template. Could we make a new dataset_base or similar that both inherit from maybe?